specification/domainservices/resource-manager/Microsoft.AAD/stable/2017-06-01/domainservices.json

```diff
@@ -311,6 +311,242 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AAD/domainServices/{domainServiceName}/replicaSets": {
+      "get": {
+        "tags": [
+          "ReplicaSets"
+        ],
+        "operationId": "ReplicaSets_ListByResourceGroup",
+        "summary": "List ReplicaSets in Resource Group (GET Resources)",
```

Comment:
What's the purpose of `(GET Resources)` part?

Reply:
I can remove that.